otter/convergence/service.py

```diff
@@ -121,7 +126,8 @@ def execute_convergence(tenant_id, group_id, log,
     log.msg('execute-convergence',
             servers=servers, lb_nodes=lb_nodes, steps=steps, now=now,
             desired=desired_group_state, active=active)
-    yield _update_active(scaling_group, active)
+    if group_status != ScalingGroupStatus.DELETING:
```

Comment:
Did you make this conditional to avoid race conditions or something?

I'm imagining a scenario where we're deleting a group and deleting all CLBs/servers, and making this conditional will mean someone watching group state will see "active" servers that aren't actually active any more, for example after they have been removed from the CLB but before they have been deleted.

It's unlikely to be an important detail, since the group is destined to be deleted anyway, but there may be strange situations where it takes a very long time to clean up a group and it may be useful to track the status through the API. Leaving the _update_active unconditional would allow this to happen, so I'm just curious about why we're conditionalizing it.